woof-distro/arm/debian/bullseye-veyron-speedy/_00build_2.conf

```diff
@@ -0,0 +1,13 @@
+#
+# use defaults (_00build.conf)
+#
+# but override these settings:
+
+## Download and include custom SFS (XDRV_INC= overrides this)
+#ADRV_SFS_URL=
+#YDRV_SFS_URL=
+#FDRV_SFS_URL=
+
+ADRV_INC=
+BUILD_CROS_IMAGE=yes
+BOOT_BOARD='veyron-speedy'
```

Comment:
Like woof-distro/x86/debian/bullseye, woof-distro/arm/debian/bullseye-veyron-speedy "inherits" from woof-distro/x86_64/debian/bullseye64. All files except DISTRO_SPECS and _00build_2.conf are symlinks. #2033 and #2234 allow this elegant reuse.